examples/remotecontent/sushichef.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+from ricecooker.chefs import SushiChef
+from ricecooker.classes.files import AudioFile
+from ricecooker.classes.files import DocumentFile
+from ricecooker.classes.licenses import get_license
+from ricecooker.classes.nodes import AudioNode
+from ricecooker.classes.nodes import DocumentNode
+from ricecooker.classes.nodes import RemoteContentNode
+from ricecooker.classes.nodes import TopicNode
+
+"""
+This example shows how to use the RemoteContentNode to create a channel that
+curates content from another channel already on Studio into a new channel.
+"""
+
+SOURCE_DOMAIN = "testdomain.org"  ## change me!
+
+original_channel_data = {
```

Comment:
As I read through this it was odd that this null-filled dict was being referenced in a class definition until I got to the bottom and saw how the values were being set before the constructor call. 

I think that this would read a bit more clearly if the values for this were passed as kwargs to the CuratedChannelChef constructor.

Reply:
The reason for this was that we need somewhere "global" to store the computed IDs that are generated during the creation of the original channel -- which would be messy to pull out later through introspection of that channel.

In terms of how they then get passed into the curated chef, I agree that arguments would be nicer. But currently [the kwargs passed into a chef constructor](https://github.com/jamalex/ricecooker/blob/d464ee9863f5807af28058be7766be17811cd7dd/ricecooker/chefs.py#L53) are then just ignored completely and not stored anywhere. There are lots of things around these ricecooker idioms I'd love to rearchitect, but I was picking my battles for this PR.

Reply:
(For now I'll add a comment here to make this clearer, though)